documentation/staging/content/quickstart/create-domain.md

```diff
@@ -117,22 +115,13 @@ weight: 3
           name: sample-domain1-cluster-cluster-1
           port: 8001
     {{% /expand %}}
+
+    b. Then apply the file using:
     ```shell
     $ kubectl apply -f /tmp/quickstart/ingress-route.yaml \
       --namespace sample-domain1-ns
     ```
 
-      b. **NOTE**: Instead of running the previous  `kubectl apply` command, you can download the ingress route YAML file to `/tmp/quickstart/ingress-route.yaml` or similar, using the following command:
```

Comment:
Based on slack discussion, should we provide a link to download the "ingress-route.yaml"? For e.g. you can download the ingress route YAML file from [here](https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/main/kubernetes/samples/quick-start/ingress-route.yaml -o /tmp/quickstart/ingress-route.yaml)..

Reply:
Agree. Done.